### ISSUE TITLE
[FF] Image toolbar's position is incorrect after pasting the image at the beginning of the document

### ISSUE THREAD

**Issue by Mgsy** [ckeditor/ckeditor5-image#174]
## Steps to reproduce
1. Go to http://localhost:8125/ckeditor5-easy-image/tests/manual/easyimage.html.
2. Upload the image.
3. Cut/copy it.
4. Paste it at the beginning of the document.

## Expected result
The image toolbar shows up below the image.

## Current result
The image toolbar shows up on the image.

## GIF
![bug_cke5](https://user-images.githubusercontent.com/20988892/35678346-41b40e62-0753-11e8-8051-0ed04b8ebff9.gif)

## Other information
OS: All platforms
Browser: Firefox
Release: Latest master

**Comment by oleq** [comment]
The funny thing is that I couldn't reproduce it in http://localhost:8125/ckeditor5-core/tests/manual/article.html. Can you, @Mgsy?

**Comment by Mgsy** [comment]
>The funny thing is that I couldn't reproduce it in http://localhost:8125/ckeditor5-core/tests/manual/article.html. Can you, @Mgsy?

Yep, I noticed that this bug doesn't occur for the default image in article sample. It looks like the issue touches only uploaded images.

**Comment by gauravjain028** [comment]
I am also having an issue with toolbar when we scroll in classic editor.
Please see the screenshot.

The below issue happens with default editor.
![Image of Toolbar issue](https://www.dropbox.com/s/4x6xpm2hngzgw39/Screen%20Shot%202018-03-13%20at%206.17.36%20PM.png?dl=0)

And when I use it in my app, I got like 
![Image of Toolbar issue in my app ](https://www.dropbox.com/s/9b4iuizbzvvoajc/Screen%20Shot%202018-03-13%20at%206.30.05%20PM.png?dl=0)

Is there any workaround this and this bug is related to this issue or should create other issue?
